config/default.yml

```diff
@@ -1,7 +1,11 @@
 listen_port: 3000
 
 db:
-    uri: insert-db-uri-here
+    user: user
+    password: password
+    host: host
+    port: port
+    database: database
```

Comment:
🎉 This is so much easier than remembering the URI format (or if you're like me, looking it up every time). 👍